config/routes.rb

```diff
@@ -7,7 +7,7 @@
   resources :users
 
   root 'search#index'
-  get '/search', to: 'artists#show'
+  get '/search', to: 'search#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.